scripts/cleanup_sandboxes.py

```diff
@@ -62,3 +63,4 @@ def get_workspaces() -> list[str]:
     for workspace in workspaces:
         if workspace not in excluded:
             trigger_delete_workflow(token=gh_pat, sandbox=workspace)
+            time.sleep(300)

```

Comment:
We maybe need a comment here to explain why we're waiting e.g.
```suggestion
            time.sleep(300) # Wait 5 min between executions to avoid an AWS concurrency issue.
```